MapboxNavigation/NavigationMapView.swift

```diff
@@ -270,8 +270,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         super.layoutSubviews()
         
         //If the map is in tracking mode, make sure we update the camera after the layout pass.
-        if (tracksUserCourse) {
-            updateCourseTracking(location: userLocationForCourseTracking, camera:self.camera, animated: false)
+        if (tracksUserCourse) {            
+            if #available(iOS 14.0, *) {
+                // Since layoutSubviews() is called too often on iOS 14.0, it leads to lags in `UserCourseView` animated updates.
+                // Workaround is to allow `UserCourseView` updates only on iOS versions lower than 14.0.
```

Comment:
If the device orientation changes or the user closes the step table, the puck will be stuck in an incorrect state until the next location update. I think that would be worth documenting as a known issue and treating as tail work.

Reply:
BTW I did some tests on iPad and iPhone Simulators and do not experience inaccurate state during device rotation. This is positive flow though. We also get some undesired `UserCourseView` shift when `updateCourseTracking(location:camera:animated:)` is called inside `layoutSubviews()`:

![output](https://user-images.githubusercontent.com/1496498/94497770-1d735600-01ad-11eb-9e03-52dc6a55a1cf.gif)

This can be taken care of in #2660.